src/sass/partials/components/_forms.scss

```diff
@@ -24,6 +24,34 @@
     }
   }
 
+  &--add {
+    @include button(setting-get('primary accent color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary color'), setting-get('primary color'));
+
+    background-image: url('/images/icons/add.svg');
```

Comment:
This may be easier for us to read if `image`, `position`, `repeat`, and `size` were all nested under `background` like you're doing with `border` below